content/news/0.10.0_release.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Release 0.10.0"
+date = 2019-03-06
+publishdate = 2018-03-06
+author = "@hsanjuan"
++++
+
+## 20190306 | Release 0.10.0
+
+Today we release 0.10.0, a release with major changes under the hood that will
+make IPFS Cluster perform significantly faster with large pinsets and less
+memory demanding.
+
+For those upgrading, this release is a mandatory step before any future
+upgrades, as it will upgrade the internal state to a new format which prepares
+the floor for the upcoming addition of an alternative CRDT-based "consensus"
```

Comment:
Should we be appropriate and use the word `convergence` ? May be this works, easy to understand this way!

Reply:
"convergence component" ?

I don't know. Maybe it should be renamed to the `state` component and refactored (replacing the current `state` one). So that state is provided by Raft or by CRDT. But for the moment the component is called "consensus".